app/serializers/v2/moves_serializer.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module V2
+  class MovesSerializer
+    include JSONAPI::Serializer
+
+    INCLUDED_ATTRIBUTES = %i[
+      additional_information
+      cancellation_reason
+      cancellation_reason_comment
+      created_at
+      date
+      date_from
+      date_to
+      move_agreed
+      move_agreed_by
+      move_type
+      reference
+      rejection_reason
+      status
+      time_due
+      updated_at
+    ].freeze
+
+    INCLUDED_FIELDS = {
+      moves: INCLUDED_ATTRIBUTES +
+        %i[profile from_location to_location prison_transfer_reason supplier additional_information],
+      profiles: ::V2::ProfileSerializer::INCLUDED_ATTRIBUTES + %i[person],
+      people: ::V2::PersonSerializer::INCLUDED_ATTRIBUTES + %i[gender ethnicity],
+      locations: ::LocationSerializer::INCLUDED_ATTRIBUTES,
+      prison_transfer_reasons: ::PrisonTransferReasonSerializer::INCLUDED_ATTRIBUTES,
+      suppliers: ::SupplierSerializer::INCLUDED_ATTRIBUTES,
+
+      # allocation: %i[to_location from_location moves_count created_at],
+    }.freeze
```

Comment:
```suggestion
INCLUDED_FIELDS = {
      moves: attributes_to_serialize.keys +
        %i[profile from_location to_location prison_transfer_reason supplier additional_information],
      profiles: ::V2::ProfileSerializer.attributes_to_serialize.keys + %i[person],
      people: ::V2::PersonSerializer.attributes_to_serialize.keys + %i[gender ethnicity],
      locations: ::LocationSerializer.attributes_to_serialize.keys,
      prison_transfer_reasons: ::PrisonTransferReasonSerializer.attributes_to_serialize.keys,
      suppliers: ::SupplierSerializer.attributes_to_serialize.keys,
    }.freeze
```

Reply:
now switched to `attributes_to_serialize.keys`